rskj-core/src/test/java/co/rsk/peg/PowpegMigrationTest.java

```diff
@@ -591,14 +596,14 @@ private void testChangePowpeg(
             if (oldPowPegFederationType == FederationType.legacyErp || oldPowPegFederationType == FederationType.p2shErp){
                 assertNotEquals(lastRetiredFederationP2SHScript, originalPowpeg.getP2SHScript());
             }
-            assertEquals(lastRetiredFederationP2SHScript, originalPowpeg instanceof ErpFederation ? ((ErpFederation) originalPowpeg).getStandardP2SHScript() : originalPowpeg.getP2SHScript());
+            assertEquals(lastRetiredFederationP2SHScript, originalPowpeg instanceof ErpFederation ? ((ErpFederation) originalPowpeg).getDefaultP2SHScript() : originalPowpeg.getP2SHScript());
```

Comment:
We should have a method to simplify this and avoid doing it everywhere. See https://github.com/rsksmart/rskj/blob/bridge-btc-tx-index-integration/rskj-core/src/main/java/co/rsk/peg/PegUtilsLegacy.java#L388-L398

Reply:
thanks!

Reply:
but that class doesnt exist yet in my code